.stylelintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  // See https://github.com/wagtail/stylelint-config-wagtail for rules.
+  extends: '@wagtail/stylelint-config-wagtail',
+  rules: {
+    // Would be valuable for strict BEM components but is too hard to enforce with legacy code.
+    'no-descending-specificity': null,
```

Comment:
Can this rule be scoped to anything outside of client only maybe? Or is there also 'legacy' code there?

https://stylelint.io/user-guide/configure/#overrides